docs/content/examples/uses-cases/folders.md

```diff
@@ -0,0 +1,56 @@
+---
+title: 'Use Cases | Folders'
+hide:
+  - toc # Hide Table of Contents for this page
+---
+## Introduction
+
+This provides a solution letting different email clients sharing the same archive folder.
```

Comment:
Does this really mean "share" the same folder (between accounts) or "share" the same configuration?

This wording might be confusing depending on what is meant indeed.